libs/chain-signatures/contract/src/tee/tee_participant.rs

```diff
@@ -183,14 +185,17 @@ impl TeeParticipantInfo {
             Some(r) => hex::encode(r.rt_mr3),
             None => return false,
         };
-        let replayed_rtmr3 = replay_rtmr(event_log.to_owned(), 3);
+        let replayed_rtmr3 = replay_rtmr(event_log.to_owned(), RTMR3_INDEX.try_into().unwrap());
```

Comment:
What type are we converting to here in the `try_into()` call? We should avoid unwraps to not cause panics.
 
```suggestion
        let replayed_rtmr3 = replay_rtmr(event_log.to_owned(), RTMR3_INDEX.try_into().unwrap());
```

Reply:
`replay_rtmr` is expecting u8.

Why not have the const `RTMR3_INDEX` be u8 instead of u64 instead?

Reply:
I see it will be more casting if you cast up from `u8` to `u64` but it will always give us a strong guarantee at compile time that the conversion never fails, and code won't panic.

It's also obvious to the reader of the code that `RTMR3_INDEX` is supposed to be a small value by setting it to a `u8`.

If you cast up to `u64` from `u8` it will be quite simple, just add `as u64` as a suffix.
```rust
 e["imr"].as_u64() == Some(RTMR3_INDEX as u64)
```